notebooks/quickstart_tutorial.ipynb

```diff
@@ -313,9 +313,15 @@
    "source": [
     "### Exploring the Cellarium CAS response\n",
     "\n",
-    "We recommending exploring the CAS response using our provided ``CASCircularTreePlotUMAPDashApp`` Dash App for a more streamlined and holistic visualization of the CAS response. The visualization is self-explanatory.\n",
+    "We recommend exploring the CAS response using our provided ``CASCircularTreePlotUMAPDashApp`` Dash App for a more streamlined and holistic visualization of the CAS response.\n",
     "\n",
-    ">**Note:** In a nutshell, the visualization shows various cell type ontology terms as colored circles. The size of the circle signifies the occurence of the term in the entire dataset (or over the chosen group of cells). The color of the circle signifies the relevance score of the term in cells over which the term was found to have any degree of relevance. You can highlight the ontology term relevance scores over the UMAP scatter plot by clicking on the circles. You can also show the terms relevant to individual cells and their scores by clicking on a cell over the UMAP scatter plot. "
+    ">**Tooltip:** The visualization displays various cell type ontology terms as colored circles in a circular dendogram. The relationships underlying this dendogram correspond to \"_is_a_\" relationships from [Cell Ontology](https://obofoundry.org/ontology/cl.html) (CL). Since these relationships are not mutually exclusive, a term can have multiple parent terms, meaning the same term can appear along different branches of the tree representation. The radius of each circle (whether it is a clade or a leaf node) signifies the occurrence of the term in the entire dataset, regardless of its relevance score. The color of the circle indicates the relevance score of the term in cells where it was found to have non-vanishing relevance.\n",
```

Comment:
```suggestion
    ">**Tooltip:** The visualization displays various cell type ontology terms as colored circles in a circular dendrogram. The relationships underlying this dendrogram correspond to \"_is_a_\" relationships from [Cell Ontology](https://obofoundry.org/ontology/cl.html) (CL). Since these relationships are not mutually exclusive, a term can have multiple parent terms, meaning the same term can appear along different branches of the tree representation. The radius of each circle (whether it is a clade or a leaf node) signifies the occurrence of the term in the entire dataset, regardless of its relevance score. The color of the circle indicates the relevance score of the term in cells where it was found to have non-vanishing relevance.\n",
```